src/Core/Authoring/ManifestReader.cs

```diff
@@ -340,9 +340,17 @@ private static List<ManifestFile> ReadFilesList(XElement xElement)
                 string target = file.GetOptionalAttributeValue("target").SafeTrim();
                 string exclude = file.GetOptionalAttributeValue("exclude").SafeTrim();
 
+                char separator = Path.DirectorySeparatorChar;
+
                 // Multiple sources can be specified by using semi-colon separated values. 
                 files.AddRange(from source in srcElement.Value.Trim(';').Split(';')
-                               select new ManifestFile { Source = source.SafeTrim(), Target = target.SafeTrim(), Exclude = exclude.SafeTrim() });
+                               select new ManifestFile 
+                               {
+                                   // Replace directory separator in the file src element to the one that is correct for the OS of the packing system.
+                                   Source = source.SafeTrim().Replace('/',separator).Replace('\u005c',separator),

```

Comment:
I'm not sure that the Windows switch is necessary - Windows can find paths that are separated by "/" just fine.

Reply:
It can handle them, but not completely correctly. It will produce nested directories.
See this example of this old version of `adobereader`. It has the scripts in `tools\tools\` rather then in `tools\`
https://chocolatey.org/packages/adobereader/2020.006.20042#files

To reproduce, change the path separator from `\` to `/` on a windows machine, run pack, and inspect the resulting nupkg.

Reply:
See: https://github.com/chocolatey/choco/issues/2166

Reply:
Also:  https://github.com/chocolatey/choco/commit/e5a02a05590f849bc57a3b55b12b88e04df7dfce